Firestore/Example/FuzzTests/FSTFuzzTestsPrincipal.mm

```diff
@@ -49,29 +61,76 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
   return 0;
 }
 
+// Retrieves the fuzzing target from the FUZZING_TARGET environment variable.
+// Default target is kNone if the environment variable is empty, not set, or
+// could not be interpreted. Should be kept in sync with FuzzingTarget.
+FuzzingTarget GetFuzzingTarget() {
+  char *fuzzing_target_env = std::getenv("FUZZING_TARGET");
+
+  if (fuzzing_target_env == NULL || strlen(fuzzing_target_env) == 0) {
+    LOG_WARN("No value provided for FUZZING_TARGET environment variable.");
+    return FuzzingTarget::kNone;
+  }
+
+  if (strcmp("NONE", fuzzing_target_env) == 0) {
+    return FuzzingTarget::kNone;
+  }
+  if (strcmp("SERIALIZER", fuzzing_target_env) == 0) {
+    return FuzzingTarget::kSerializer;
+  }
+  // Value did not match any target.
```

Comment:
Optional: consider omitting this comment, the log message seems clear enough.

Reply:
ok.